app/v2/models/responses/hateoas/HateoasResponse.scala

```diff
@@ -19,15 +19,18 @@ package v2.models.responses.hateoas
 import config.Constants
 import v2.models._
 import v2.models.responses.hateoas.HateoasResponse.prefix
-
 import java.time.OffsetDateTime
 
 object HateoasResponse {
 
   lazy val prefix =
-    if (routing.routes.DeparturesRouter.submitDeclaration().url.startsWith(Constants.Context)) ""
+    if (
+      routing.routes.DeparturesRouter
```

Comment:
my lack of understanding here but why do we need this bit of code? I thought that if it started with context it would be calling .urlWithContext rather than .url

Reply:
Also this is a val...and you have implemented an OR here, so if one of them starts with context and the other does not, it will be duplicated for the one that already contains it

Reply:
removed or condition as suggested by Dan

Reply:
@drnaylor-oc yeah makes sense - In my PR I am already making that modification btw @sonisn1988